arelle/plugin/validate/EDINET/Constants.py

```diff
@@ -1,8 +1,17 @@
 """
 See COPYRIGHT.md for copyright information.
 """
+from enum import Enum
+
 from arelle.ModelValue import qname
 
+class DeiFormType(Enum):
+    CORPORATE_FORMS = [
+        '第二号の四様式',  # Form 2-4
+        '第二号の七様式',  # Form 2-7
+        '第三号様式'  # Form 3
+    ]
+
```

Comment:
An Enum is implemented more like this:

```suggestion
class DeiFormType(Enum):
    FORM_2_4 = '第二号の四様式'
    FORM_2_7 = '第二号の七様式'
    FORM_3 = '第三号様式'
    
    def isCorporateForm(self) -> bool:
        return self in CORPORATE_FORMS
        
CORPORATE_FORMS = {DeiFormType.FORM_2_4, ... }

```
`LinkbaseType` is an example.